README.md

```diff
@@ -12,23 +12,29 @@ As a geocacher, I want to be able to see a list of cache points that I can navig
 
 ### Example 
 
-  
-
 **Given**: A feed of geocache points data are available 
 
-  
+**When**: The user/service selects a particular cache point
 
-**When**: The user/service selects a particular cache point 
+**When**: The user/service shows the latitude/longitude of said cache point 
 
-  
+**Then**: The user’s/service’s cache point will be saved with the correct latitude/longitude. 
 
-**When**: The user/service shows the latitude/longitude of said cache point 
+### Example
 
-  
+**Given**: The user is selects a latitude(90) and a longitude(100)
 
-**Then**: The user’s/service’s cache point will be saved with the correct latitude/longitude. 
+**When**: The user selects a geo-cache point
 
-  
+**Then**: The user’s/service’s cache point will be saved with the selected latitude/longitude.
+
+### Example
+
+**Given**: The user is selects a latitude(pizza) and a longitude(burgers)
+
+**When**: The user selects a geo-cache point that is invalid
+
+**Then**: The application will throw an error and ask for the user to reenter their location
```

Comment:
Uses TDD to take from requirements to test specific use cases and cases where the user provides inaccurate data. 
- References: 
[Products not Projects](https://martinfowler.com/articles/microservices.html)
[Technical Debt](https://martinfowler.com/bliki/TechnicalDebt.html)